lib/methods/export.js

```diff
@@ -115,56 +151,63 @@ module.exports = self => {
         .toArray();
     },
 
-    getRelatedDocsIds(manager, doc, relatedTypes) {
-      // Use `doc.type` for pages to get the actual schema of the corresponding page type.
-      const schema = manager.schema || self.apos.modules[doc.type].schema;
+    getRelatedIdsBySchema({
+      doc, schema, type = 'relationship', relatedTypes, recursion = 0
+    }) {
+      return schema.flatMap(field => {
+        const fieldValue = doc[field.name];
+        const shouldRecurse = recursion < MAX_RECURSION;
```

Comment:
```suggestion
        const shouldRecurse = recursion <= MAX_RECURSION;
```

Reply:
Hum since the recursion starts at 0, if I put `<=` I'll have 11 call of the function.
But in a way you're right, the first call isn't a recursion 🤔 
Hum the universe is so complex, I need to let my body float in the universe, nothing makes sense.
let me modify this line of code

Reply:
yeah but a 1 recursion means that we are passing a 2nd time in the function.
to me, "recursion" = "re-called".

that's what's been done in apiRoutes